internal/http/request.go

```diff
@@ -67,6 +81,18 @@ func (r *Request) SetHeader(key string, value string) {
 	r.request.Header.Set(key, value)
 }
 
+// GetHeader returns the value of the header requested
+func (r *Request) GetHeader(key string) string {
+	return r.request.Header.Get(key)
+}
+
+// DelHeader returns the value of the header requested
+func (r *Request) DelHeader(key string) {
```

Comment:
How about `DeleteHeader`?  No strong opinion here, I like the symmetry with the underlying API also.

Reply:
Yeah, Del is what's under the hood which is why I left it.  Delete feels better, but so does `Header.Del` for phrasing.